account_invoice_start_end_dates/models/account_move_line.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2016 Akretion, Alexis de Lattre <alexis.delattre@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    start_date = fields.Date('Start Date', index=True)
```

Comment:
The attribute string is redundant. String parameter equal to name of variable (attribute-string-redundant)
```suggestion
    start_date = fields.Date(index=True)
```